src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -272,6 +276,17 @@ public async override System.Threading.Tasks.Task RunTaskAsync ()
 					cmd.Add (GetFullPath (assetDir));
 				}
 			}
+
+			if (AdditionalAndroidAssetPaths != null) {
+				for (int i = 0; i < AdditionalAndroidAssetPaths.Length; i++) {
+					var assetDir = AdditionalAndroidAssetPaths [i].ItemSpec.TrimEnd ('\\');
+					if (!string.IsNullOrWhiteSpace (assetDir) && Directory.Exists (assetDir)) {
+						cmd.Add ("-A");
+						cmd.Add (GetFullPath (assetDir));
+					}
```

Comment:
As per the bug we were seeing with `WorkingDirectory` and `<Aapt2Link/>`, I think this needs to be:
```suggestion
					var assetDir = GetFullPath (AdditionalAndroidAssetPaths [i].ItemSpec.TrimEnd ('\\'));
					if (Directory.Exists (assetDir)) {
						cmd.Add ("-A");
						cmd.Add (assetDir);
					} else {
						LogDebugMessage ("asset directory did not exist: " + assetDir);
					}
```

Reply:
done